GettingStarted/CheatSheet.txt

```diff
@@ -0,0 +1,155 @@
+         a. fix any errors
+	 b. re-do until errors are all corrected
+  4. from menu bar:    Kernel --> Restart and Clear All Cells
+  5. hit "save" in the notebook editor to make sure edits are captured.
+
+
+
+# Commit changes to local branch:
+git status
+git add <filename>      # if there is a file ("<filename>") to add...
+git commit -am "Add a fairly detailed comment on the changes you made here"  # check in changes
+git status
+
+
+# Push updates to origin on github:
+git push origin project/helloworld/douglasleetucker
+
+
+# Make "pull" request:
+Go to https://github.com/LSSTScienceCollaborations/StackClub/pulls and click on green "New pull request" button.
+
+
+# After getting e-mail saying "pull" request has been approved, perform a "merge":
+# (Note:  there may be some redundancy here.  I was following the workflow described in
+#         the youtube video listed under "General git tutorial stuff" in the "Other useful
+#         git commands" section below.)
+git checkout master     # Switch from project/helloworld/douglasleetucker back to master branch (locally)
+git pull origin master  # Make sure local master branch is up-to-date with remote origin master branch on github
+git branch --merged     # Check to see if project/helloworld/douglasleetucker has been merged locally
+git merge project/helloworld/douglasleetucker      # Merge project/helloworld/douglasleetucker with local master branch
+git branch --merged     # Verify the merge worked
+git push origin master  # Push local master back to remote origin master on github
+git branch -d project/helloworld/douglasleetucker  # Delete local version of project/helloworld/douglasleetucker
+git push origin --delete project/helloworld/douglasleetucker  # Delete remote version of project/helloworld/douglasleetucker on github
+git branch -a           # Verify that the local and remote versions of project/helloworld/douglasleetucker have been deleted
+
+
+# Start with a new branch to address other issues:
+git status              # Always good to check...
+git checkout -b issue#11-hello-world-douglasleetucker   # Create a new branch and move to it...
+git branch              # Confirm you are now working on that branch
+# ... and so on...
+
+
+-----------------------------------------------------------
+Other useful git commands:
+
+git --version
+git config --global user.email "dtucker@fnal.gov"  # create/update your e-mail address in the local git configuration
+git config --global user.name "DouglasLeeTucker"   # create/update your username in the local git configuration
+git config --list
+git config --help
+git help config
+git log
+git diff
+
+
+General git tutorial stuff:
+  https://www.youtube.com/watch?v=HVsySz-h9r4
+  https://medium.com/@christo8989/what-college-students-should-learn-about-git-6bbf6eaac39c
+  https://github.com/drphilmarshall/GettingStarted
+  
+
+-----------------------------------------------------------
+Useful Slack Channels:
+  dm-lsp-users
+  dm-newbies
```

Comment:
I don't see the "dm-newbies" channel - does a "newbie" have to request access to it?